### ISSUE TITLE
FR: "Reset all settings to default" button

### ISSUE THREAD

**Issue by ShlomoCode** [ejbills/DockDoor#142]
We need one in the settings.

**Comment by ejbills** [comment]
Simple to do, we can write a method that iterates through every default key and reset it.

Something like this (AI generated so I'm not sure if this syntax is correct, think of it as pseudocode):
`import Defaults

func resetDefaultsToDefaultValues() {
    for key in Defaults.keys {
        // Check if there is a default value specified in the property wrapper
        if let defaultValue = key.defaultValue {
            // Reset the key to its default value
            Defaults[key] = defaultValue
        } else {
            // If no default value is specified, remove the key
            Defaults.remove(key)
        }
    }
}`

**Comment by ShlomoCode** [comment]
Yes, they have a built-in reset method: https://github.com/sindresorhus/Defaults#reset-keys-to-their-default-values

Note that we must not reset the `launched' key, otherwise the user will suddenly get a "Welcome" window... 😀
